internal/slicer/report.go

```diff
@@ -32,7 +32,7 @@ type Report struct {
 // provided root path.
 func NewReport(root string) *Report {
 	return &Report{
-		Root:    root,
+		Root:    filepath.Clean(root),
```

Comment:
There's an issue here if we look at the Add behavior below. That HasPrefix below is not correct unless there's necessarily a slash added to the end of the root path here, otherwise "/foo/bar" is also a prefix of "/foo/bare", for example. There are multiple ways to fix this, but enforcing the slash here, potentially after the Clean, might be a cheap one.

Reply:
You are right, I have added the "/" after clean and a test.